code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -18,8 +18,20 @@
 	hunger_enabled = 0//Until automated eating mechanics are enabled, disable hunger for hostile mobs
 	var/shuttletarget = null
 	var/enroute = 0
+
+	// Vars to help find targets
 	var/list/targets = list()
 	var/attacked_times = 0
+	var/list/target_type_validator_map = list()
+	var/lowest_health = INFINITY // Max you can get
+	var/atom/T = null

```

Comment:
This _has_ to be renamed. `T` is not descriptive, which is why variables named like that only exist within current scope, where their use can be quickly dicerned.